css/styles.css

```diff
@@ -65,130 +66,131 @@ input#textbox{
 	justify-content: center;*/
 }
 
-.listdiv ul{
-	margin: 0;
-	padding: 0;
-	background-color: solid gray;
-	list-style: none;
-	/*justify-content: center;
+.listdiv ul {
+  margin: 0;
+  padding: 0;
+  background-color: solid gray;
+  list-style: none;
+  /*justify-content: center;
 	display: flex;*/
 }
 
-ul li{
-/*	cursor: crosshair;*/
-	position: center;
-	padding: 12px 8px 12px 40px;
-	background: #eee;
-	font-size: 20px;
+ul li {
+  /*	cursor: crosshair;*/
+  position: center;
+  padding: 12px 8px 12px 40px;
+  background: #eee;
+  font-size: 20px;
 }
 
-ul li:hover{
-	background: #ddd;
+ul li:hover {
+  background: #ddd;
 }
 
 /*Strikes through data when its checkbox is checked*/
-input[type=checkbox]:checked + label{
-	text-decoration: line-through;
+input[type=checkbox]:checked+label {
+  text-decoration: line-through;
 }
 
-.submitbutton{
-	margin: 0;
-	width: 300px;
-	align-self: center;
+.submitbutton {
+  margin: 0;
+  width: 300px;
+  align-self: center;
 }
 
-.allButton:hover{
-	filter: brightness(50%);
+.allButton:hover {
+  filter: brightness(50%);
```

Comment:
Careful for using `filter`. Some browsers don't support this well yet. 

You can use `opacity` instead.